.github/config.yml

```diff
@@ -4,22 +4,36 @@
 
 # Comment to be posted to on first time issues
 newIssueWelcomeComment: |
-  Thanks for opening your first issue here! Please follow the issue template to help us help you 👍🎉😄
-  If you have screenshots to share demonstrating the issue, that's really helpful! 📸 You can [make a gif](https://www.cockos.com/licecap/) too!
+  Thanks for opening your first issue! This space is [protected by our Code of Conduct](https://publiclab.org/conduct) - and we're here to help.  
+  Please follow the issue template to help us help you 👍🎉😄   
+  If you have screenshots [or a gif](https://publiclab.org/gifs/) to share demonstrating the issue, that's really helpful! 📸
+  Do join our [Gitter channel](https://gitter.im/publiclab/publiclab) for some brainstorming discussions. 
 # Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
 
 # Comment to be posted to on PRs from first time contributors in your repository
 newPRWelcomeComment: |
- Thanks for opening this pull request! `Dangerbot` will test out your code and reply in a bit with some pointers and requests.
- There may be some errors, **but don't worry!** We're here to help! 👍🎉😄
+ Thanks for opening this pull request!  This space is [protected by our Code of Conduct](https://publiclab.org/conduct) - and we're here to help.
+ `Dangerbot` will test out your code and reply in a bit with some pointers and requests.
+ Also please refer [here](https://github.com/publiclab/PublicLab.Editor#installation) for installation help 💿    
+ There may be some errors, **but don't worry!** We'll work through them with you! 👍🎉😄    
+ It would be great if you can tell us your Twitter handle so we can thank you properly?
```

Comment:
We may also want to highlight that we'd like the version number to be bumped in `package.json` (pointing to the file URL) and that we'd like the changes built into the `/dist/` folder as well?

This is awesome!

Reply:
Alright I took care of it [here](https://github.com/publiclab/PublicLab.Editor/commit/4922436531646df0c6ab038f7809c14c7e1fcdcd). Thanks!